docs/source/management/monitoring/overview.md

```diff
@@ -2,55 +2,86 @@
 
 ## Architecture
 
-ScyllaDB [exposes](https://monitoring.docs.scylladb.com/stable/reference/monitoring-apis.html) its metrics in Prometheus format. 
-{{productName}} leverages this capability in Kubernetes environments to provide a comprehensive monitoring solution for your ScyllaDB clusters.
-{{productName}} supports deploying and managing a complete monitoring stack for your ScyllaDB clusters using the
-[ScyllaDBMonitoring](../../api-reference/groups/scylla.scylladb.com/scylladbmonitorings.rst) custom resource.
+ScyllaDB [exposes](https://monitoring.docs.scylladb.com/stable/reference/monitoring-apis.html) its metrics in the Prometheus format. 
+{{productName}} provides the [`ScyllaDBMonitoring`](../../api-reference/groups/scylla.scylladb.com/scylladbmonitorings.rst) custom resource
+that allows you to set up a complete monitoring stack for your ScyllaDB clusters based on the following components:
+
+- [**Prometheus**](https://prometheus.io) for metrics collection and alerting (along with scraping and alerting rules targeting ScyllaDB instances).
+- [**Grafana**](https://grafana.com) for metrics visualization (with pre-configured dashboards for ScyllaDB).
+
+:::{include} diagrams/monitoring-overview.mmd
+:::
 
 :::{note}
-ScyllaDBMonitoring CRD is still experimental. The API is currently in version `v1alpha1` and may change in future versions.
+The `ScyllaDBMonitoring` CRD is still in its `v1alpha1` version, yet it is considered stable and ready for production use, with
+the following caveats:
+- `spec.components.grafana.exposeOptions` and `spec.components.prometheus.exposeOptions` are deprecated and will be removed in the next API version,
+- the **Managed** mode is deprecated; the **External** mode for Prometheus is likely to be the only supported mode in the next API version.
```

Comment:
Huh? When has that been decided? Why is the managed prometheus supposed to be removed?

Reply:
Discussed offline with both of you. My point is that the cost of making a mistake here isn't high; if users bump into issues with the external mode, they always have the option to go with the managed mode for the time being. And keeping one endorsed way as a clear default will leave the user with one less decision to make (=easier to use).

Also, we always have the option to un-deprecate.

@rzetelskik with your permission I suggest that we keep the direct "deprecation" wording.